src/users/containers/utils/user-roles-util.ts

```diff
@@ -72,7 +97,10 @@ export class UserRolesUtil {
    * @param response - the response object from PRD.
    */
   public static doesRoleDeletionExist(response) {
-    return response.roleDeletionResponse && AppUtils.propsExist(response, ['roleDeletionResponse']);
+    return (
+      response.roleDeletionResponse &&
```

Comment:
strange way of formatting conditionals - does this pass yarn lint?

Reply:
yup, but will revert it to a single line.

Reply:
I don't really mind if you don't revert it - I was just concerned that there are some strict whitespace rules in our tslint profile

Reply:
Reverted to single line in commit:
https://github.com/hmcts/rpx-xui-manage-organisations/commit/f13b896f6d2d616fe1b63f082089d9c50d05af97